### PR TITLE
Updated Section Leaders

### PR DESCRIPTION
**Pull request by patfedigan**
Be proud to bear the rainbow

That's all I know I can contribute

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -37,15 +37,15 @@
```
Comment (mckinleyrebecca) [inline index.html:37]: Ay yo Patrick, how did you get permission to push to a branch?? I can't :(

Comment (patfedigan) [reply index.html:37]: Ay yo Rebecca!! I had to fork it first to my own repo. Can @yabberyabber make this open so anyone can contribute? Is that even allowed on github?

Comment (mckinleyrebecca) [reply index.html:37]: Ahh, gotcha, thanks!

Comment (yabberyabber) [reply index.html:37]: Can I make it open and still have gh-pages be a protected branch?

Comment (yabberyabber) [reply index.html:37]: Figured it out.  @mckinleyrebecca can you resolve conflict and I'll merge?

Comment (mckinleyrebecca) [reply index.html:37]: Done!

**Pull request merged**